src/main/java/shef/servlets/NewLiveStreamServlet.java

```diff
@@ -25,19 +26,22 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet responsible for creating new livestreams. */
+/** Servlet responsible for creating new LiveStream entities
+    and storing them in Datastore. */
 @WebServlet("/new-live-stream")
 public class NewLiveStreamServlet extends HttpServlet {
 
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String recipeKey = request.getParameter("recipe-selection");
-    String link = request.getParameter("live-stream-link");
-
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Entity liveStreamEntity = new Entity("LiveStream");
-    liveStreamEntity.setProperty("recipe-key", recipeKey);
-    liveStreamEntity.setProperty("link", link);
-
+    Map<String, String[]> paramMap = request.getParameterMap();
+    if (paramMap.get("duration")[0].equals("P0D")) {
```

Comment:
I'm a little confused where this value comes from and don't see it used elsewhere or defined as a default anywhere.

Reply:
Defined and clarified (added as static final string).